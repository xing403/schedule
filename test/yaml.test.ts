import YAML from 'js-yaml'
import { expect, it } from 'vitest'

it('yaml string parse', () => {
  expect(YAML.load(`
      key: 'date-time'
      args:
        format: 'YYYY-MM-DD HH:mm:ss'
  `)).toMatchSnapshot()
})
