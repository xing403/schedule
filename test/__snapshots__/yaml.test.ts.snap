// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`yaml string parse 1`] = `
{
  "args": {
    "format": "YYYY-MM-DD HH:mm:ss",
  },
  "key": "date-time",
}
`;
